core/server/index.js

```diff
@@ -108,6 +112,14 @@ function init(options) {
             });
 
         return new GhostServer(parentApp);
+    }).then(function (_ghostServer) {
+        ghostServer = _ghostServer;
+
+        // scheduling can trigger api requests, that's why we initialize the module after the ghost server creation
+        // scheduling module can create x schedulers with different adapters
+        return scheduling.init(_.extend(config.scheduling, {apiUrl: config.url + config.urlFor('api')}));
+    }).then(function () {
+        return ghostServer;
```

Comment:
I could test installing Ghost via npm. Tell me if you still have any concerns, even when they are small.